### ISSUE TITLE
Повышение удобства рекомендации тегов

### ISSUE THREAD

**Issue by 7nik** [stalkerg/anime-pictures#11]
Было бы неплохо отображать на картинке уже рекомендованные, но ещё не проверенные теги. Так пользователям будет проще искать, что они забыли. ИМХО, ещё такая система будет более стимулирующая, так как пользователи будут сразу видеть результат их роботы.

**Comment by iDolmatov** [comment]
Сейчас такую идею реализовали на пикабу, только для меня она остаётся немного мудрёной.

Желательно, чтобы список непроверенных тегов не добавлял много лишнего места.

Какой сейчас средний максимум рекомендованных тегов?
2-4 тега ещё можно отображать дополнительно, а сотню уже сложнее.

Как сейчас применяются рекомендации одно тега разными пользователями?
Например, можно при наведении на тег видеть всех пользователей с рекомендациями.
А модератору можно показывать пользователя и цифру.
Если будет полезно, то может и на пре переползёт фича, когда рекомендации плюсуются. А не как сейчас, что вроде предложил тег, а он оказывается был и ты его не заметил. Визуально тег остался.

Как с цветовым оформлением?
Синий цвет или какой?
Если тег добавляется, то может не писать ок, а небольшую анимацию добавления тега замутить?
Например, тег добавлен - появление, тег уже есть - лёгкое мигание.

**Comment by 7nik** [comment]
Я это вижу всё тот же список тегов, просто некоторые из них помечены другим цветом, например, синим, и для модератора имеют иконки принятия/отклонения вместо выделения и удаления тега.

Больше тридцати тегов практически не бывает (ведь норма норма это 20-35 всех тегов на картинку, а тут часть уже проставлена), да и накладок тоже практически не бывает, но кто первый того и лавры.

Имхо чужие рекомендованные теги не следует показывать, там не редко та ещё чушь проскакивает, а как я уже говорил накладки случаются очень редко, да и принимаются/отклоняются они в течение суток.

**Comment by iDolmatov** [comment]
> чужие рекомендованные теги не следует показывать
Вот это важное замечание, которое теряется в первом сообщении. Думал, что должны видеть все.

**Comment by 7nik** [comment]
Немного поправил описание.

**Comment by stalkerg** [comment]
Так, это мега идея. Полностью поддерживаю. (только сейчас всё осознал)
Короче как только так сразу.

**Comment by 7nik** [comment]
За пол года ни только, ни сразу так и не наступили :)
Юзерскрипт для модерирования рекомендованных тегов.
[AP moderate tags.user.js.txt](https://github.com/stalkerg/anime-pictures/files/2374963/AP.moderate.tags.user.js.txt)
А выглядит это вот так
![screenshot_5](https://user-images.githubusercontent.com/29704098/45423136-07b09800-b69b-11e8-834b-33cefca3e3bf.png)
но оформить можно и немного по другому.
